google-cloud-datastore/src/main/java/com/google/cloud/datastore/DatastoreImpl.java

```diff
@@ -543,8 +543,10 @@ public List<Key> reserveIds(Key... keys) {
 
   com.google.datastore.v1.ReserveIdsResponse reserveIds(
       final com.google.datastore.v1.ReserveIdsRequest requestPb) {
-    Span span = traceUtil.startSpan(TraceUtil.SPAN_NAME_RESERVEIDS);
```

Comment:
Nit: Do you want to clean up this if it's not used anywhere? https://github.com/googleapis/java-datastore/blob/e7636fdf3bd582f15ba7511ec12dfba7a289a132/google-cloud-datastore/src/main/java/com/google/cloud/datastore/TraceUtil.java#L39

Reply:
I plan to clean-up all the OpenCensus code once I'm code-complete for Otel - will send a follow-up PR.